GameServerApp/Program.cs

```diff
@@ -19,70 +19,70 @@ private static void Main(string[] args)
         {
             _logger = LoggerProvider.GetLogger();
 
-            var parsedArgs = ArgsOptions.Parse(args);
-
-            parsedArgs.GameInfoJson = LoadConfig(
-                parsedArgs.GameInfoJsonPath,
-                parsedArgs.GameInfoJson,
-                Encoding.UTF8.GetString(Resources.GameInfo));
-
-            var gameServerBlowFish = "17BLOhi6KZsTtldTsizvHg==";
-            var gameServerLauncher = new GameServerLauncher(
-                parsedArgs.ServerPort,
-                parsedArgs.GameInfoJson,
-                gameServerBlowFish);
-
-#if DEBUG
-            var configGameServerSettings = GameServerConfig.LoadFromJson(LoadConfig(
-                parsedArgs.GameServerSettingsJsonPath,
-                parsedArgs.GameServerSettingsJson,
-                Encoding.UTF8.GetString(Resources.GameServerSettings)));
-            if (configGameServerSettings.AutoStartClient)
-            {
-                var leaguePath = configGameServerSettings.ClientLocation;
-                if (Directory.Exists(leaguePath))
-                {
-                    leaguePath = Path.Combine(leaguePath, "League of Legends.exe");
-                }
-                if (File.Exists(leaguePath))
+            ArgsOptions.Parse(args).WithParsed((parsedArgs) => {
```

Comment:
Just cast this to a variable if possible, higher indentation level is generally a bad practice

Reply:
Not to mention it changes the entire diff even though technically you modified just a single line

Reply:
Fixed